src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -141,9 +143,9 @@ export class DownloadOptionPanel extends SimpleComponent {
     }
 
     render() {
-        const {groupKey, cutoutSize, help_id, children, style, title, tbl_id, dlParams, dataTag} = this.props;
+        const {groupKey, cutoutSize, help_id, children, style, title, tbl_id, dlParams, dataTag, labelWidth} = this.props;
         const {mask, email, enableEmail} = this.state;
-        const labelWidth = 110;
+        const lblWidth = labelWidth|| 110;
```

Comment:
No need to define this awkward `lblWidth` variable.  Just use `labelWidth` and default it to 110 at its declaration line.. i.e.
`...  dataTag, labelWidth=110} = this.props;`